alibi_detect/od/llr.py

```diff
@@ -245,7 +245,8 @@ def logp(self, dist, X: np.ndarray, return_per_feature: bool = False, batch_size
         Log probabilities.
         """
         logp_fn = partial(dist.log_prob, return_per_feature=return_per_feature)
-        return predict_batch(X, logp_fn, batch_size=batch_size)
+        # TODO: TBD: can this be any of the other types from predict_batch? i.e. tf.Tensor or tuple
```

Comment:
I'm not sure about this one. I'd have guessed that `predict_batch` won't return `tf.Tensor` here since we don't explicitly set `dtype` (so `return_np=True`), but it looks like `tuple` might be possible? Hopefully @arnaudvl has more input!